octofree/build-with-tui.sh

```
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Build script that handles the multi-directory structure
+
+echo "🐙 Building Octofree Docker Image with TUI Monitor..."
+echo ""
+
+# Change to parent directory to include both octofree and bubbletea in build context
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)